ic.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+
+const assert = require('assert');
+const fs = require('fs');
+const rp = require('request-promise-native');
+const username = process.env.CIRCLE_TOKEN;
+const password = '';
+const auth = 'Basic ' + new Buffer(username + ':' + password).toString('base64');
+
+var projectName = JSON.parse(fs.readFileSync('./package.json')).name;
+//Override project name in the command line
+if(process.argv.length === 3) projectName = process.argv[2]
+
+const firstCheckToken = {
+    method: 'GET',
+    uri: 'https://circleci.com/api/v1.1/me',
+    headers: {
+      'Authorization': auth
+    },
+    json: true,
+    resolveWithFullResponse: false,
+};
+
+const secondCreateProject = {
+    method: 'POST',
+    uri: `https://circleci.com/api/v1/project/Zenika/${projectName}/follow`,
+    headers: {
+      'Authorization': auth
+    },
+    json: true, // Automatically parses the JSON string in the response
```

Comment:
The comment doesn't sound too useful, as this info is easily discovered in the `request` doc.